ui/job-view/pushes/Push.jsx

```diff
@@ -29,6 +29,17 @@ import { RevisionList } from './RevisionList';
 const watchCycleStates = ['none', 'push', 'job', 'none'];
 const platformArray = Object.values(thPlatformMap);
 
+const fetchTestManifestsArtifact = async (project, revision) => {
+  let taskNameToManifests = {};
+  const rootUrl = 'https://firefox-ci-tc.services.mozilla.com';
```

Comment:
I can look into that but this *only* makes sense for Firefox test jobs since the artifact is only created for Hg repos.
I might want to prevent the unnecessary call for non-Firefox repos.